### PR TITLE
Now also adds commands from the cpp frames

### PR DESCRIPTION
**Pull request by jasperdewinther**

### CODE REVIEW

File: module/cli_controller.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import json
 import threading
 import queue
+import inspect
 
+import common.frames
 from client.comm import BaseComm
 
-from module.command_node import CommandNode
+from modules.swarm_controller.module.command_node import CommandNode
```
Comment (Timmotus) [inline module/cli_controller.py]: this should not change to a longer import

File: module/cli_controller.py [hunk 2]
```diff
@@ -46,58 +48,86 @@ def load_tree(self):
...
+        commands = [frame for frame in frames if frame[1].DESCRIPTION ]
+        
+        #add all commands to root node
+        for command in commands:
+            target_node = CommandNode(
+                command[0][5:],
+                parameter_list=inspect.getfullargspec(command[1].set_data).annotations,
+                command_info=command[1].DESCRIPTION
+            )
+            print("initialised " + command[0][5:])
```
Comment (Timmotus) [inline module/cli_controller.py]: this should probably not be here

File: module/cli_controller.py [hunk 3]
```diff
@@ -46,58 +48,86 @@ def load_tree(self):
...
+                current_node[new_node.name] = new_node
+                current_node = new_node
+
+        # After all the missing links in the tree are made, add the command
+        target_node = CommandNode(
+            json_command["target"],
+            parameter_list=json_command["parameters"],
+            command_info=json_command["info"]
+        )
+        print("initialised " + json_command["target"].lower())
```
Comment (Timmotus) [inline module/cli_controller.py]: should also probably not be here

**Review by Timmotus — CHANGES_REQUESTED** [review]

File: module/cli_controller.py (hunk 2 above)
Comment (jasperdewinther) [reply module/cli_controller.py]: Wouldn't it be a good indicator if something went wrong?

File: module/cli_controller.py (hunk 3 above)
Comment (jasperdewinther) [reply module/cli_controller.py]: Wouldn't it be a good indicator if something went wrong?

File: module/cli_controller.py (hunk 2 above)
Comment (jasperdewinther) [reply module/cli_controller.py]: NVM i think it might be better to add these things in a test program

File: module/test.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+#print(classes)
+
+print(FrameDisplayFilledRectangle().comment)
+
+#for i in classes:
+#    print(i[1])
+#    comment = i[1]().get_comment()
+#    if comment:
+#        print(comment)
```
Comment (Timmotus) [inline module/test.py]: what purpose does this file serve?

File: module/cli_controller.py (hunk 3 above)
Comment (Timmotus) [reply module/cli_controller.py]: then we should show what went wrong instead of what went right

File: module/test.py (hunk 4 above)
Comment (jasperdewinther) [reply module/test.py]: Dont know how that got there

**Review by Timmotus — APPROVED** [review]

**Pull request merged**